mappings/net/minecraft/structure/Structure.mapping

```diff
@@ -6,22 +6,115 @@ CLASS net/minecraft/class_3499 net/minecraft/structure/Structure
+		ARG 7 offsetY
+		ARG 8 offsetZ
+	METHOD method_15174 record (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2338;ZLnet/minecraft/class_2248;)V
+		ARG 2 pos
+		ARG 3 size
+		ARG 4 recordEntities
+		ARG 5 blockToIgnore
 	METHOD method_15175 toTag (Lnet/minecraft/class_2487;)Lnet/minecraft/class_2487;
 		ARG 1 tag
+	METHOD method_15176 mirrorAndRotate (Lnet/minecraft/class_243;Lnet/minecraft/class_2415;Lnet/minecraft/class_2470;Lnet/minecraft/class_2338;)Lnet/minecraft/class_243;
+		ARG 0 size
+		ARG 1 mirror
+		ARG 2 rotation
+		ARG 3 pos
+	METHOD method_15177 readBlocks (Lnet/minecraft/class_2499;Lnet/minecraft/class_2499;)V
+		ARG 1 palettesTag
+		ARG 2 blocksTag
+	METHOD method_15178 placeAndNotifyListeners (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;Lnet/minecraft/class_3492;)V
+		ARG 1 world
+		ARG 2 pos
+		ARG 3 placementData
+	METHOD method_15179 placeEntities (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;Lnet/minecraft/class_2415;Lnet/minecraft/class_2470;Lnet/minecraft/class_2338;Lnet/minecraft/class_3341;)V
+		ARG 1 world
+		ARG 2 structurePos
+		ARG 3 mirror
+		ARG 4 rotation
+		ARG 5 relativePos
+		ARG 6 area
+	METHOD method_15180 mirrorAndRotateBox (Lnet/minecraft/class_3492;Lnet/minecraft/class_2338;Lnet/minecraft/class_3492;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2338;
+		ARG 1 placementData1
+		ARG 2 pos1
+		ARG 3 placementData2
+		ARG 4 pos2
 	METHOD method_15181 getAuthor ()Ljava/lang/String;
-	METHOD method_15182 place (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;Lnet/minecraft/class_3492;)V
+	METHOD method_15182 placeAndCreateBoundingBox (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;Lnet/minecraft/class_3492;)V
+		ARG 1 world
+		ARG 2 pos
+		ARG 3 placementData
 	METHOD method_15183 fromTag (Lnet/minecraft/class_2487;)V
 		ARG 1 tag
 	METHOD method_15184 createDoubleListTag ([D)Lnet/minecraft/class_2499;
+	METHOD method_16186 mirrorBoundingBox (Lnet/minecraft/class_2470;IILnet/minecraft/class_3341;Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;)V
+		ARG 1 rotation
+		ARG 2 offsetX
+		ARG 3 offsetZ
+		ARG 4 boundingBox
 	METHOD method_16187 calculateBoundingBox (Lnet/minecraft/class_3492;Lnet/minecraft/class_2338;)Lnet/minecraft/class_3341;
 		ARG 1 placementData
 		ARG 2 pos
+	METHOD method_16445 getInfosForBlock (Lnet/minecraft/class_2338;Lnet/minecraft/class_3492;Lnet/minecraft/class_2248;)Ljava/util/List;
+		ARG 1 pos
+		ARG 2 placementData
 	METHOD method_16446 process (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;Lnet/minecraft/class_3492;Ljava/util/List;)Ljava/util/List;
 		ARG 0 world
 		ARG 1 pos
 		ARG 2 placementData
 		ARG 3 blockInfos
+	METHOD method_17916 getEntity (Lnet/minecraft/class_1936;Lnet/minecraft/class_2487;)Ljava/util/Optional;
+		ARG 0 world
+		ARG 1 tag
+	METHOD method_20532 bulkUpdateBlockStates (Lnet/minecraft/class_1936;ILnet/minecraft/class_251;III)V
+		ARG 0 world
+		ARG 1 flags
+		ARG 3 minX
+		ARG 4 minY
+		ARG 5 minZ
```

Comment:
Why are there just min coordinates and not max coordinates? Does it update starting at those coordinates up to the edge of the structure? If yes, `updateCorner` or `updateCornerBlockStates` with parameter names `startX` `startY` `startZ` would be clearer.